statsmodels/base/tests/test_penalized.py

```diff
@@ -705,17 +705,20 @@ def _initialize(cls):
 
     def test_params_table(self):
         # override inherited because match is not good except for params and predict
+        # The cov_type in GLMPenalized and in TheilGLS are not the same
+        # both use sandwiches but TheilGLS sandwich is not HC
+        # relative difference in bse up to 7%
```

Comment:
in places where there is potential ambiguity about what is a multi-line comment vs separate comments, I've started using the pattern:

```
# some comment [...]
#  continuation of the comment gets an extra space
# Separate comment no extra space
```